software.md

```diff
@@ -43,7 +43,7 @@ Here you can find links to software related to RPM such as frontends, packaging
 * [perl-RPM2](http://search.cpan.org/dist/RPM2/) provides an object-oriented interface to RPM facilities for Perl.
 * [perl-RPM4](http://search.cpan.org/~tvignaud/RPM4/) provides an object-oriented interface to RPM facilities for Perl.
 * [RPM-Specfile](http://search.cpan.org/dist/RPM-Specfile/) is a Perl extension for creating RPM specfiles.
-* [Ruby-RPM](http://rubyforge.org/projects/ruby-rpm/) provides Ruby bindings to RPM.
+* [Ruby-RPM](https://github.com/hansode/ruby-2.1.x-rpm) provides Ruby bindings to RPM.
```

Comment:
Wait, is this the correct link? It seems like a completely different project...

Reply:
Nice catch. Yeah, I should have looked into the content not only the name of the project. Looks like the original project has more or less vanished from the internet. I guess we'll just remove the entry.

Reply:
Yup, there seem to be at least two other similar projects:
https://github.com/rpm-software-management/rpm-web/pull/22
https://github.com/dmacvicar/ruby-rpm
But neither looks like being in active development... So yeah, let's just remove the entry then.